admin-dev/themes/new-theme/js/types/global.d.ts

```diff
@@ -74,33 +74,33 @@ type FetchResponse = Record<string, any>;
 
 type OptionsObject = FetchResponse;
 
-type VTags = Vue & {
+type VTags = {
   add: (tag: any) => void;
   tag: any;
 };
 
-type VTagsInput = Vue & {
+type VTagsInput = {
   value: any;
 };
 
-type VCheckbox = Vue & {
+type VCheckbox = {
   $data: VCheckboxDatas;
 };
 
-type VCheckboxDatas = Vue & {
+type VCheckboxDatas = {
   checked: boolean;
 };
 
 interface SelectorsMap extends Record<string, string> {
   [key: string]: string;
 }
 
-type VTransModal = Vue & {
+type VTransModal = {
   showModal: () => void;
   $once: any;
 };
 
-type VPrincipal = Vue & {
+type VPrincipal = {
```

Comment:
I can't comment on the following files so I level the comment here, Why drop the `.d` in the name, did the conventions change?